### PR TITLE
Create service account screen

### PR DESCRIPTION
**Pull request by jinapurapu**
**How it looks:**
Screenshot from 2022-04-22 16-57-34

Closes https://github.com/miniohq/engineering/issues/727

### CODE REVIEW

**Review by cniackz — CHANGES_REQUESTED** [review]
Please squash the commits to only one

**Review by cniackz — DISMISSED** [review]
LGTM

**Review by adfost — DISMISSED** [review]

File: portal-ui/src/screens/Console/Account/AddServiceAccountScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+  const [accessKey, setAccessKey] = useState<string>("");
+  const [secretKey, setSecretKey] = useState<string>("");
+  const [isRestrictedByPolicy, setIsRestrictedByPolicy] =
+    useState<boolean>(false);
+  const [addCredentials, setAddCredentials] = useState<boolean>(false);
+ const [newServiceAccount, setNewServiceAccount] = useState<NewServiceAccount | null>(null);
+  const [showPassword, setShowPassword] = useState<boolean>(false);
+
+ useEffect(() => {
+ console.log("Displaying prompt - NewServiceAccount:", newServiceAccount)
```
Comment (prakashsvmx) [inline portal-ui/src/screens/Console/Account/AddServiceAccountScreen.tsx:124]: Please remove the console logs (3) in the component

File: portal-ui/src/screens/Console/Account/AddServiceAccountScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+            setAddSending(false);
+            setErrorSnackMessage(err);
+          });
+      } else {
+        api
+          .invoke("POST", `/api/v1/service-accounts`, {
+            policy: policyDefinition,
+          })
+          .then((res) => {
+            console.log("From API2newSA:", res);
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Account/AddServiceAccountScreen.tsx:154]: Please remove console.log

**Review by bexsoft — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/Console/Account/AddServiceAccountScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+                        }}
+                        label={"Customize Credentials"}
+                        
+                      />
+                      </Grid>
+                       <Grid item xs={12}>
+                      {addCredentials && (
+                          <Grid container item spacing ="20">
+                        <Grid item xs={12}>
+                          <div className={classes.stackedInputs} >
```
Comment (prakashsvmx) [inline portal-ui/src/screens/Console/Account/AddServiceAccountScreen.tsx:258]: Please make the spacing below this input consistent with other forms.

**Review by dvaldivia — DISMISSED** [review]

**Review by dvaldivia — DISMISSED** [review]

**Review by dvaldivia — APPROVED** [review]

**Review by bexsoft — APPROVED** [review]

**Pull request merged**